jormungandr/src/main.rs

```diff
@@ -201,14 +202,11 @@ fn start_services(bootstrapped_node: BootstrappedNode) -> Result<(), start_up::E
     let enclave = Enclave::from_vec(leader_secrets);
 
     {
-        use tokio::prelude::*;
-
+        let leadership_logs = leadership_logs.clone();
+        let enclave = leadership::Enclave::new(enclave.clone());
         let fragment_pool = fragment_pool.clone();
-        let block_task = block_task.clone();
         let blockchain_tip = blockchain_tip.clone();
-
-        let enclave = leadership::Enclave::new(enclave.clone());
-        let stats_counter = stats_counter.clone();
```

Comment:
except for `leadership_logs` that needed to be cloned, there is no useful changes here. Why did you re-organise the order of these lines?

Reply:
I've changed order so it matches order of usage few lines below. It's easier to follow what comes from where. I've touched this block anyway.

Reply:
but now you are breaking git history with non necessary code changes.

Reply:
I can revert it, but I don't agree that fighting entropy is unnecessary. It's not worth a separate PR and I've already touched this block, it'll be dirty anyway.